src/main/java/TestNotificationClient.java

```diff
@@ -161,4 +166,18 @@ private static void createPostRequest(NotificationClient client) throws IOExcept
             System.out.println(response);
         }
     }
+
+    private static void getTemplate(NotificationClient client) throws IOException, NotificationClientException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+        System.out.println("Enter the template id: ");
+        String templateId = reader.readLine();
+        if (templateId == null || templateId.isEmpty()) {
```

Comment:
is stringutils available in this project? StringUtils.isEmpty()

Reply:
No, StringUtils is not available in the project.
It would be a good option, however, I did not want to introduce new dependencies - and it is used this way in other places in the same class.